Color.js

```diff
@@ -38,4 +38,10 @@ class Color {
       this.hexCode = this.makeRandomColor();
     }
   }
+
+  lockIconValue() {
+    return this.isLocked
+    ? "lock"
+    : "lock_open";
+  }
 }
```

Comment:
Add this as a method of the color class. This will dry up our code for the toggle function.